.env.dev

```diff
@@ -13,13 +13,17 @@ PLIVO_PHONE_NUMBER=1555555555
 OTP_DIGITS=6
 OTP_TTL_MINUTES=5
 WALLET_URL=http://localhost:3000
-MAUTIC_URL=https://go.gooddollar.org
-MAUTIC_TOKEN=""
+MAUTIC_URL=https://go.gooddollar.org/api
+MAUTIC_TOKEN=NjdlZmE4NmEwNjA3OTBiNTQ1ZmVjMDEyMzgxZjkxYTQyZDc4N2M5MThiOTFjN2Q0Y2U5ODhjNjU1NzNhYTU4ZA
```

Comment:
you can't put any tokens or secrets in github!

Reply:
done